src/Microsoft.Health.Fhir.Api/Features/Operations/Reindex/ReindexJobWorkerBackgroundService.cs

```diff
@@ -30,11 +30,11 @@ public ReindexJobWorkerBackgroundService(ReindexJobWorker reindexJobWorker, IOpt
             _reindexJobConfiguration = reindexJobConfiguration.Value;
         }
 
-        protected override async Task ExecuteAsync(CancellationToken cancellationToken)
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
Why is this cancellation token renamed but none of the others are?

Reply:
[https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.backgroundservice.executeasync?view=dotnet-plat-ext-5.0#Microsoft_Extensions_Hosting_BackgroundService_ExecuteAsync_System_Threading_CancellationToken](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.backgroundservice.executeasync?view=dotnet-plat-ext-5.0#Microsoft_Extensions_Hosting_BackgroundService_ExecuteAsync_System_Threading_CancellationToken)_
because for this particular case we use method which define name of cancellation token as stoppingToken.

---
In reply to: [617670289](https://github.com/microsoft/fhir-server/pull/1858#discussion_r617670289) [](ancestors = 617670289)